src/test/scala/FastaTests.scala

```diff
@@ -125,6 +125,68 @@ class FastaTests extends FunSuite {
         parsedFile.appendLines(map("sequence").grouped(70).mkString("\n"))
       }
     }
+  }
+
+  test("FASTA lines parsing") {
+
+    val crap = Seq(
+      "hola fasta! lalala",
+      "oh no, no soy fasta"
+    )
+
+    val fasta1 = FASTA(
+      header(FastaHeader(">1 hola")) ::
+      sequence(FastaSequence("ATCACCCACTTTACATTTCACACACCCCTTTACAC")) ::
+      *[AnyDenotation]
+    )
+
+    val fasta2 = FASTA(
+      header(FastaHeader(">2 hola")) ::
+      sequence(FastaSequence("ATATACCCACACCCCGGTCAT")) ::
+      *[AnyDenotation]
+    )
+
+    val emptyFasta = FASTA(
+      header(FastaHeader(">3 nothing")) ::
+      sequence(FastaSequence("")) ::
+      *[AnyDenotation]
+    )
+
+    assert {
+      fasta.parseFastaDropErrors( (crap ++ fasta1.lines).iterator ).toList ==
+        List(fasta1)
+    }
+
+    // NOTE fails
+    // assert {
+    //   fasta.parseFastaDropErrors( (emptyFasta.lines ++ fasta1.lines).iterator ).toList ==
+    //     List(emptyFasta, fasta1)
+    // }
```

Comment:
This is as it's supposed to be: FASTA with empty sequence is ignored. I wrote about it in https://github.com/ohnosequences/fastarious/pull/14#issuecomment-200364865. Did I misunderstand you and it should be kept?